scripts/Makefile

```diff
@@ -0,0 +1,36 @@
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+# Install CRDs into a cluster
+install: generate-crds
+	kubectl apply -f config/crd
+
+# Uninstall CRDs from a cluster
+uninstall: generate-crds
+	kubectl delete config/crd
+
+# Generate CRDs
+generate-crds: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd
+
+# download controller-gen if necessary
+controller-gen:
+ifeq (, $(shell which controller-gen))
+	@{ \
+	set -e ;\
+	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\g
+	cd $$CONTROLLER_GEN_TMP_DIR ;\
+	go mod init tmp ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
+	rm -rf $$CONTROLLER_GEN_TMP_DIR ;\
+	}
+CONTROLLER_GEN=$(GOBIN)/controller-gen
+else
+CONTROLLER_GEN=$(shell which controller-gen)
+endif
```

Comment:
so this is the kubebuilder-generated Makefile, yeah?

I think I'd prefer to do a couple things with this:

1) Convert it into a proper shell script instead of the Makefile monstrosity that it currently is. For example, you can make an `ensure_controller_gen` Bash function like this:

https://github.com/aws/aws-service-operator-k8s/blob/mvp/scripts/lib/aws.sh#L17

to replace the `controller-gen` Makefile target above.

2) Make the script accept a CLI argument of the AWS service to generate CRDs for (that way we have a single script that we can run for any AWS service controller)

Reply:
>> so this is the kubebuilder-generated Makefile, yeah?
Yes I liked the way it was defined there so took part of it relevant and modified.

Sure makes sense to have bash script instead. Will apply those changes in next commit. thanks.